### PR TITLE
Assignment 1 - Ryan Golden

### PR DESCRIPTION
**Pull request by Xionxiii**
https://icy-desert-0d4f2cf1e.3.azurestaticapps.net

### CODE REVIEW

**Review by dpeterson8 — COMMENTED** [review]
Looks good, website running good as well

**Comment by grahamna** [comment]
I like you transition effects between pages.

File: wordle-web/src/views/GameView.vue [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  <v-card color="grey-darken-4">
+    <v-card-title> Wordlinator </v-card-title>
+    <v-card-subtitle>This is a game about words</v-card-subtitle>
+    <v-card-text>
+      Here is some placeholder text for the game, lorem ipsum solte delore sit amet siv percum.
+    </v-card-text>
+    <v-card-actions>
+      <nav>
+        <v-btn variant="outlined" to="/" color="arabicaMint">Return to Home</v-btn>
+      </nav>
```
Comment (mmwoodfo) [inline wordle-web/src/views/GameView.vue:11]:
```suggestion
        <v-btn variant="outlined" to="/" color="arabicaMint">Return to Home</v-btn>
```
You don't need to wrap this in a nav tag

**Review by mmwoodfo — COMMENTED** [review]
Good job Ryan!

**Pull request closed without merging**